### PR TITLE
nsxiv: update to 30

### PR DESCRIPTION
**Pull request by WitherCubes**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/nsxiv/patches/no-rebuild-options.patch [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Preventing rebuild of options.o when installing
 --- a/Makefile
 +++ b/Makefile
-@@ -106,7 +106,7 @@
+@@ -81,7 +81,7 @@
  		rm -f "$$dir/nsxiv.png"; \
  	done
```
Comment (paper42) [inline srcpkgs/nsxiv/patches/no-rebuild-options.patch:7]: this patch refresh should not be necessary

Comment (WitherCubes) [reply srcpkgs/nsxiv/patches/no-rebuild-options.patch:7]: But the patch offsets -25 lines. Won't that be a problem?

Comment (paper42) [reply srcpkgs/nsxiv/patches/no-rebuild-options.patch:7]: no

File: srcpkgs/nsxiv/template [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 make_install_target=install-all
 makedepends="libX11-devel imlib2-devel giflib-devel libexif-devel libXft-devel
  libwebp-devel freetype-devel fontconfig-devel"
 short_desc="Neo (or New or Not) Simple (or Small or Suckless) X Image Viewer"
 maintainer="Sanjay Pavan <withercubes@protonmail.com>"
 license="GPL-2.0-or-later"
-homepage="https://github.com/nsxiv/nsxiv"
-distfiles="https://github.com/nsxiv/nsxiv/archive/v${version}.tar.gz"
-checksum="564a17d921172336f7cd463bf313e39f84e4bcf4fdc789326f4112f848aad039"
+homepage="https://codeberg.org/nsxiv/nsxiv"
```
Comment (classabbyamp) [inline srcpkgs/nsxiv/template:13]:
```suggestion
homepage="https://codeberg.org/nsxiv/nsxiv"
changelog="https://codeberg.org/nsxiv/nsxiv/raw/branch/master/CHANGELOG.md"
```

**Comment by paper42** [comment]
please squash the commits

**Pull request merged**